domains/authorization/Authorization.API/API/Authorization/[Features]/Internal/GetConsentForUserCommand.cs

```diff
@@ -97,6 +96,12 @@ public async Task<GetConsentForUserCommandResult> Handle(GetConsentForUserComman
             userRepository.Update(user);
         }
 
+        var consentReceivedOrganizationIds = await organizationRepository.Query()
+            .Where(org => org.Id == organization.Id)
+            .SelectMany(org => org.OrganizationReceivedConsents)
+            .Select(consent => consent.ConsentGiverOrganizationId)
+            .ToListAsync(cancellationToken);
```

Comment:
Skal man have SelectMany ind i en separat Select?

Reply:
I did so for clarity.